myst_nb/converter.py

```diff
@@ -188,6 +197,7 @@ def myst_to_notebook(
     :param raw_directive: the name of the directive to search for containing raw cells
     :param add_source_map: add a `source_map` key to the notebook metadata,
         which is a list of the starting source line number for each cell.
+    :param: context: passthrough Sphinx Builder Environment
```

Comment:
re-write this so you don't pass the sphinx builder to this function, which should be independent of sphinx

Reply:
@chrisjsewell we need to use the `srcdir` from the `builder environment` to adjust the `path`. Would you prefer I extract `srcdir` and pass that instead of the passing the builder `context`?

Reply:
@chrisjsewell I have removed context passing and added file_srcdir to support the file import option in 12a11a3

Reply:
I need to use more than one context based variable so I have renamed the context to file_context to indicate it is used by the :file: option to code-cell in 3207b34. That will make it easier to remove if this get's implemented elsewhere.